### PR TITLE
fix running

### LINKED ISSUE #2217: Getting started docker, python and ipython version mismatch error.
Hello, I was trying to run the getting started container and the IPython kernel was not able to start because the ipython was throwing an error about the version mismatch with current python version.

Downgrading IPython to 7.10 fixed the issue of starting IPython kernel but the notebook also seems to throw an error related to undefined "ErrorCode". 

Logs:
```
Starting indy_pool ... done
Starting getting_started ... done
Attaching to indy_pool, getting_started
indy_pool    | 2020-07-14 07:44:46,150 CRIT Set uid to user 1000
getting_started | [I 07:44:47.064 NotebookApp] Serving notebooks from local directory: /home/indy
getting_started | [I 07:44:47.064 NotebookApp] The Jupyter Notebook is running at:
getting_started | [I 07:44:47.064 NotebookApp] http://4b3f24d43692:8888/?token=e8e9d47034b730135f196e830b6a29e58473c2d77f3c5963
getting_started | [I 07:44:47.064 NotebookApp]  or http://127.0.0.1:8888/?token=e8e9d47034b730135f196e830b6a29e58473c2d77f3c5963
getting_started | [I 07:44:47.064 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
getting_started | [W 07:44:47.068 NotebookApp] No web browser found: could not locate runnable browser.
getting_started | [C 07:44:47.068 NotebookApp] 
getting_started |     
getting_started |     To access the notebook, open this file in a browser:
getting_started |         file:///home/indy/.local/share/jupyter/runtime/nbserver-1-open.html
getting_started |     Or copy and paste one of these URLs:
getting_started |         http://4b3f24d43692:8888/?token=e8e9d47034b730135f196e830b6a29e58473c2d77f3c5963
getting_started |      or http://127.0.0.1:8888/?token=e8e9d47034b730135f196e830b6a29e58473c2d77f3c5963
getting_started | [I 07:44:52.108 NotebookApp] 302 GET /?token=e8e9d47034b730135f196e830b6a29e58473c2d77f3c5963 (10.0.0.1) 0.68ms
getting_started | [I 07:44:55.758 NotebookApp] Kernel started: 5fc8f92f-ae09-4ae1-aa8c-e20c7f521f90
getting_started | Traceback (most recent call last):
getting_started |   File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
getting_started |     "__main__", mod_spec)
getting_started |   File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
getting_started |     exec(code, run_globals)
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel_launcher.py", line 15, in <module>
getting_started |     from ipykernel import kernelapp as app
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel/__init__.py", line 2, in <module>
getting_started |     from .connect import *
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel/connect.py", line 13, in <module>
getting_started |     from IPython.core.profiledir import ProfileDir
getting_started |   File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 42, in <module>
getting_started |     """)
getting_started | ImportError: 
getting_started | IPython 7.10+ supports Python 3.6 and above.
getting_started | When using Python 2.7, please install IPython 5.x LTS Long Term Support version.
getting_started | Python 3.3 and 3.4 were supported up to IPython 6.x.
getting_started | Python 3.5 was supported with IPython 7.0 to 7.9.
getting_started | 
getting_started | See IPython `README.rst` file for more information:
getting_started | 
getting_started |     https://github.com/ipython/ipython/blob/master/README.rst
getting_started | 
getting_started | 
getting_started | [I 07:44:58.758 NotebookApp] KernelRestarter: restarting kernel (1/5), new random ports
getting_started | Traceback (most recent call last):
getting_started |   File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
getting_started |     "__main__", mod_spec)
getting_started |   File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
getting_started |     exec(code, run_globals)
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel_launcher.py", line 15, in <module>
getting_started |     from ipykernel import kernelapp as app
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel/__init__.py", line 2, in <module>
getting_started |     from .connect import *
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel/connect.py", line 13, in <module>
getting_started |     from IPython.core.profiledir import ProfileDir
getting_started |   File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 42, in <module>
getting_started |     """)
getting_started | ImportError: 
getting_started | IPython 7.10+ supports Python 3.6 and above.
getting_started | When using Python 2.7, please install IPython 5.x LTS Long Term Support version.
getting_started | Python 3.3 and 3.4 were supported up to IPython 6.x.
getting_started | Python 3.5 was supported with IPython 7.0 to 7.9.
getting_started | 
getting_started | See IPython `README.rst` file for more information:
getting_started | 
getting_started |     https://github.com/ipython/ipython/blob/master/README.rst
getting_started | 
getting_started | 
getting_started | /usr/local/lib/python3.5/dist-packages/jupyter_client/manager.py:358: FutureWarning: Method cleanup(connection_file=True) is deprecated, use cleanup_resources(restart=False).
getting_started |   FutureWarning)
getting_started | [I 07:45:01.763 NotebookApp] KernelRestarter: restarting kernel (2/5), new random ports
getting_started | Traceback (most recent call last):
getting_started |   File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
getting_started |     "__main__", mod_spec)
getting_started |   File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
getting_started |     exec(code, run_globals)
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel_launcher.py", line 15, in <module>
getting_started |     from ipykernel import kernelapp as app
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel/__init__.py", line 2, in <module>
getting_started |     from .connect import *
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel/connect.py", line 13, in <module>
getting_started |     from IPython.core.profiledir import ProfileDir
getting_started |   File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 42, in <module>
getting_started |     """)
getting_started | ImportError: 
getting_started | IPython 7.10+ supports Python 3.6 and above.
getting_started | When using Python 2.7, please install IPython 5.x LTS Long Term Support version.
getting_started | Python 3.3 and 3.4 were supported up to IPython 6.x.
getting_started | Python 3.5 was supported with IPython 7.0 to 7.9.
getting_started | 
getting_started | See IPython `README.rst` file for more information:
getting_started | 
getting_started |     https://github.com/ipython/ipython/blob/master/README.rst
getting_started | 
getting_started | 
getting_started | [I 07:45:04.771 NotebookApp] KernelRestarter: restarting kernel (3/5), new random ports
getting_started | Traceback (most recent call last):
getting_started |   File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
getting_started |     "__main__", mod_spec)
getting_started |   File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
getting_started |     exec(code, run_globals)
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel_launcher.py", line 15, in <module>
getting_started |     from ipykernel import kernelapp as app
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel/__init__.py", line 2, in <module>
getting_started |     from .connect import *
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel/connect.py", line 13, in <module>
getting_started |     from IPython.core.profiledir import ProfileDir
getting_started |   File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 42, in <module>
getting_started |     """)
getting_started | ImportError: 
getting_started | IPython 7.10+ supports Python 3.6 and above.
getting_started | When using Python 2.7, please install IPython 5.x LTS Long Term Support version.
getting_started | Python 3.3 and 3.4 were supported up to IPython 6.x.
getting_started | Python 3.5 was supported with IPython 7.0 to 7.9.
getting_started | 
getting_started | See IPython `README.rst` file for more information:
getting_started | 
getting_started |     https://github.com/ipython/ipython/blob/master/README.rst
getting_started | 
getting_started | 
getting_started | [I 07:45:07.779 NotebookApp] KernelRestarter: restarting kernel (4/5), new random ports
getting_started | Traceback (most recent call last):
getting_started |   File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
getting_started |     "__main__", mod_spec)
getting_started |   File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
getting_started |     exec(code, run_globals)
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel_launcher.py", line 15, in <module>
getting_started |     from ipykernel import kernelapp as app
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel/__init__.py", line 2, in <module>
getting_started |     from .connect import *
getting_started |   File "/usr/local/lib/python3.5/dist-packages/ipykernel/connect.py", line 13, in <module>
getting_started |     from IPython.core.profiledir import ProfileDir
getting_started |   File "/usr/local/lib/python3.5/dist-packages/IPython/__init__.py", line 42, in <module>
getting_started |     """)
getting_started | ImportError: 
getting_started | IPython 7.10+ supports Python 3.6 and above.
getting_started | When using Python 2.7, please install IPython 5.x LTS Long Term Support version.
getting_started | Python 3.3 and 3.4 were supported up to IPython 6.x.
getting_started | Python 3.5 was supported with IPython 7.0 to 7.9.
getting_started | 
getting_started | See IPython `README.rst` file for more information:
getting_started | 
getting_started |     https://github.com/ipython/ipython/blob/master/README.rst
getting_started | 
getting_started | 
getting_started | [W 07:45:10.787 NotebookApp] KernelRestarter: restart failed
getting_started | [W 07:45:10.787 NotebookApp] Kernel 5fc8f92f-ae09-4ae1-aa8c-e20c7f521f90 died, removing from map.
getting_started | [W 07:45:55.847 NotebookApp] Timeout waiting for kernel_info reply from 5fc8f92f-ae09-4ae1-aa8c-e20c7f521f90
getting_started | [E 07:45:55.850 NotebookApp] Error opening stream: HTTP 404: Not Found (Kernel does not exist: 5fc8f92f-ae09-4ae1-aa8c-e20c7f521f90)
getting_started | [W 07:45:56.879 NotebookApp] 404 GET /api/kernels/5fc8f92f-ae09-4ae1-aa8c-e20c7f521f90/channels?session_id=b0179227831742908a15305fb014da05 (10.0.0.1): Kernel does not exist: 5fc8f92f-ae09-4ae1-aa8c-e20c7f521f90
getting_started | [W 07:45:56.891 NotebookApp] 404 GET /api/kernels/5fc8f92f-ae09-4ae1-aa8c-e20c7f521f90/channels?session_id=b0179227831742908a15305fb014da05 (10.0.0.1) 14.60ms referer=None
getting_started | [W 07:45:58.904 NotebookApp] Replacing stale connection: 5fc8f92f-ae09-4ae1-aa8c-e20c7f521f90:b0179227831742908a15305fb014da05
^CGracefully stopping... (press Ctrl+C again to force)
Stopping getting_started ... done
Stopping indy_pool       ... done
```

Getting started notebook error:

```
---------------------------------------------------------------------------
PoolLedgerConfigAlreadyExistsError        Traceback (most recent call last)
<ipython-input-1-34cc7413584a> in run()
     20     try:
---> 21         await pool.create_pool_ledger_config(pool_['name'], pool_['config'])
     22     except IndyError as ex:

/usr/local/lib/python3.5/dist-packages/indy/pool.py in create_pool_ledger_config(config_name, config)
     38                         c_config,
---> 39                         create_pool_ledger_config.cb)
     40 

/usr/lib/python3.5/asyncio/futures.py in __iter__(self)
    360             self._blocking = True
--> 361             yield self  # This tells Task to wait for completion.
    362         assert self.done(), "yield from wasn't used with future"

/usr/lib/python3.5/asyncio/tasks.py in _wakeup(self, future)
    295         try:
--> 296             future.result()
    297         except Exception as exc:

/usr/lib/python3.5/asyncio/futures.py in result(self)
    273         if self._exception is not None:
--> 274             raise self._exception
    275         return self._result

PoolLedgerConfigAlreadyExistsError: 

During handling of the above exception, another exception occurred:

NameError                                 Traceback (most recent call last)
<ipython-input-1-34cc7413584a> in <module>
    731     loop = asyncio.new_event_loop()
    732     asyncio.set_event_loop(loop)
--> 733     loop.run_until_complete(run())
    734     time.sleep(1)  # FIXME waiting for libindy thread complete

/usr/lib/python3.5/asyncio/base_events.py in run_until_complete(self, future)
    385             raise RuntimeError('Event loop stopped before Future completed.')
    386 
--> 387         return future.result()
    388 
    389     def stop(self):

/usr/lib/python3.5/asyncio/futures.py in result(self)
    272             self._tb_logger = None
    273         if self._exception is not None:
--> 274             raise self._exception
    275         return self._result
    276 

/usr/lib/python3.5/asyncio/tasks.py in _step(***failed resolving arguments***)
    239                 result = coro.send(None)
    240             else:
--> 241                 result = coro.throw(exc)
    242         except StopIteration as exc:
    243             self.set_result(exc.value)

<ipython-input-1-34cc7413584a> in run()
     21         await pool.create_pool_ledger_config(pool_['name'], pool_['config'])
     22     except IndyError as ex:
---> 23         if ex.error_code == ErrorCode.PoolLedgerConfigAlreadyExistsError:
     24             pass
     25     pool_['handle'] = await pool.open_pool_ledger(pool_['name'], None)

NameError: name 'ErrorCode' is not defined
```

### PR DESCRIPTION
**Pull request by ken5scal**
Resolve #2217

### CODE REVIEW

**Comment by whackur** [comment]
I have an error with install packages.

```
Get:17 http://archive.ubuntu.com/ubuntu xenial-backports/main amd64 Packages [7942 B]
Get:18 http://archive.ubuntu.com/ubuntu xenial-backports/universe amd64 Packages [9084 B]
Fetched 16.6 MB in 4s (3406 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package ipython=
ERROR: Service 'jupyter' failed to build: The command '/bin/sh -c apt-get update -y && apt-get install -y       wget    python3.5        python3-pip     python-setuptools       ipython==7.9    ipython-notebook        apt-transport-https     ca-certificates software-properties-common' returned a non-zero code: 100
```

also I tried with Trouble Shooting Guide.
I tried to execute for build with
```
docker-compose build --no-cache
```

**Review by whackur — DISMISSED** [review]

File: docs/getting-started/getting-started.dockerfile [hunk 1]
```diff
@@ -8,7 +8,6 @@ RUN apt-get update -y && apt-get install -y \
 	python3.5 \
 	python3-pip \
 	python-setuptools \
-	ipython \
 	ipython-notebook \
```
Comment (jovfer) [inline docs/getting-started/getting-started.dockerfile:11]: should `ipython-notebook` be moved as well?

**Review by jovfer — DISMISSED** [review]

File: docs/getting-started/getting-started.dockerfile (hunk 1 above)
Comment (ken5scal) [reply docs/getting-started/getting-started.dockerfile:11]: sorry for the late reply. That may be required. let me fix

Comment (ken5scal) [reply docs/getting-started/getting-started.dockerfile]: just fixed @jovfer

**Review by jovfer — APPROVED** [review]

**Pull request merged**